packages/component/src/types.ts

```diff
@@ -19,7 +19,8 @@ import { ComponentContainer } from './component_container';
 
 export const enum InstantiationMode {
   LAZY = 'LAZY', // Currently all components are LAZY in JS SDK
```

Comment:
```suggestion
  LAZY = 'LAZY', // Currently most components are LAZY in JS SDK
```

EAGER may also use a comment too